gazebo_plugins/src/gazebo_ros_planar_move.cpp

```diff
@@ -52,6 +52,17 @@ namespace gazebo
         sdf->GetElement("robotNamespace")->Get<std::string>();
     }
 
+    enable_horizontal_movement_ = true;
+    if (!sdf->HasElement ("enableHolizontalMovement"))
```

Comment:
Woops, ho**r**izontal :wink:

Both the X and Y axes are in the horizontal plane, so this parameter can't really "disable horizontal movement". How about renaming it to something more descriptive, such as `enableYAxis`?

Reply:
That makes sense. I addressed your comments. Please have a look at it.